jetty-core/jetty-http2/jetty-http2-common/src/main/java/org/eclipse/jetty/http2/internal/HTTP2Flusher.java

```diff
@@ -466,12 +466,18 @@ public void dump(Appendable out, String indent) throws IOException
     @Override
     public String toString()
     {
-        return String.format("%s[window_queue=%d,frame_queue=%d,processed/pending=%d/%d]",
-            super.toString(),
-            getWindowQueueSize(),
-            getFrameQueueSize(),
-            processedEntries.size(),
-            pendingEntries.size());
+        try (AutoLock ignored = lock.tryLock())
+        {
+            String held = lock.isHeldByCurrentThread() ? "" : "?";
+            return String.format("%s[%s:windowQueue=%d,frameQueue=%d,processed/pending=%d/%d]",
```

Comment:
I think the `:` should only be part of the string when the lock is held.

Reply:
The problem is that then the formatting string becomes `%swindowQueue`, which is difficult to parse visually and IJ marks it with a wiggle underline because it thinks it's a misspelled "swindow". 🤷🏼‍♂️